lib/gateways/applications-api.ts

```diff
@@ -179,3 +181,24 @@ export const getStats = async (): Promise<Array<Stat> | null> => {
     return null;
   }
 };
+
+export const getApplicationHistory = async (
+  id: string,
+  req: any
+): Promise<ActivityHistoryPagedResult | null> => {
+  try {
+    const url = `${process.env.ACTIVITY_HISTORY_API}/activityhistory?targetId=${id}&pageSize=100`;
+
+    const cookies = cookie.parse(req.headers.cookie ?? '');
+    const parsedToken = cookies['hackneyToken'];
+
+    const { data } = await axios.get(url, {
+      headers: {
+        Authorization: 'Bearer ' + parsedToken,
```

Comment:
For the other Hackney APIs, we used x-api-key?

Reply:
dunno, but this works.. i can test it